interface/patient_file/summary/demographics.php

```diff
@@ -1199,11 +1202,19 @@ function filterActiveIssues(array $i): array
                     endif;
 
                     $sectionRenderEvents = $ed->dispatch(new SectionEvent('primary'), SectionEvent::EVENT_HANDLE);
+                    $sectionRenderEvents->addCard(new DemographicsViewCard($result, $result2, ['dispatcher' => $ed]));
+                    if (!$GLOBALS['hide_billing_widget']) {
+                        $sectionRenderEvents->addCard(new BillingViewCard($pid, $insco_name, $result['billing_note'], $result3, ['dispatcher' => $ed]));
+                    }
+                    $sectionRenderEvents->addCard(new InsuranceViewCard($pid, ['dispatcher' => $ed]));
                     $sectionCards = $sectionRenderEvents->getCards();
 
+                    // if anyone wants to render anything before the patient demographic list
+                    $GLOBALS["kernel"]->getEventDispatcher()->dispatch(new RenderEvent($pid), RenderEvent::EVENT_SECTION_LIST_RENDER_BEFORE, 10);
```

Comment:
@sjpadgett I had a conflict here with the Weno plugin.  I had pulled out demographics, billing and insurance into their own class objects to make the code logic easier to follow.  That however, changes up how this event fires.  I want to make sure I didn't break anything here with what your PR did here. If you can let me know that this won't impact you, I'd appreciate it.

Reply:
@adunsulag If you resolved the conflict I don't see an issue. It made better sense to me that the before event fired before billing widget. Too be honest I'd rather follow you lead and add as a new card. Sadly Sherwin wrote the included weno widget before cards and used the options widget with expand/collapse so I could rewrite I'm just not that invested.
If you need/want to revert event place let me know so I can push in my PR.

BTW: from what I see so far, nice work...

Reply:
Nope, if this works for you, I'll keep it as is as the conflict is now resolved.